homeassistant/components/switch/config_flow.py

```diff
@@ -0,0 +1,33 @@
+"""Config flow for Switch integration."""
+from __future__ import annotations
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import callback
+from homeassistant.helpers import selector
+
+from .const import DOMAIN
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required("entity_id"): selector.selector({"entity": {"domain": "switch"}}),
```

Comment:
Maybe we should only allow entity_registry id here, not entity_id as a string?

Reply:
That would require an extension of our selector spec.